src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -1046,6 +1090,19 @@ private bool ShouldExecuteSyntaxNodeActions(AnalysisScope analysisScope)
             return false;
         }
 
+        private bool ShouldExecuteOperationActions(AnalysisScope analysisScope)
+        {
+            foreach (var analyzer in analysisScope.Analyzers)
+            {
+                if (this.OperationActionsByAnalyzerAndKind.ContainsKey(analyzer))
+                {
+                    return true;
+                }
+            }
+
+            return false;
```

Comment:
OK. I'm muzzy about exactly when closures get introduced--apparently referring to a local is not sufficient to force generating one. I guess it isn't needed if there are no stores to the local(?)